proposals/NNNN-Function-to-control-the-priority-of-audio-and-video-when-switching-between-apps.md

```diff
@@ -0,0 +1,137 @@
+# Function to control the priority of audio and video when switching between apps
+
+* Proposal: [SDL-NNNN](NNNN-Function-to-control-the-priority-of-audio-and-video-when-switching-between-apps.md)
+* Author: [Akihiro Miyazaki (Nexty)](https://github.com/Akihiro-Miyazaki)
+* Status: **Awaiting review**
+* Impacted Platforms: [ Core ]
+
+## Introduction
+
+The aim of this proposal is to add a function to SDL Core. It allows OEMs to customize the status transition of audio and video when apps are switched. OEMs can control the status transition of apps when those are switched by customized app status transition rule that is in additional configuration file.
+
+## Motivation
+
+Switching apps is common SDL behavior. However, the app status transition rule is defined by the SDL Core, and OEMs can not customize the priority of audio and video when switching apps. Therefore, in some cases, it is difficult for OEMs to meet the specifications that they want to implement. For this reason, we propose adding a function which OEMs can set the priority control of audio and video according to the rules of status transition.
+
+## Proposed solution
+
+We propose adding new configuration file to the SDL Core that for defining customized the rule of status transition for audio and video by OEMs. Also we propose management rule of the file. OEMs can meet the specifications they want to implement by customizing and maintaining the configuration file.
+
+Setting rules of configuration files are below.
+
+- When the SDL is launched, the SDL Core loads two configration files, defaut configration and customized one by the OEM. Then the SDL Core selects one of the status transition rule with each situation.
```

Comment:
```suggestion
- When the SDL is launched, SDL Core loads two configuration files, the default configuration and the customized one by the OEM. Then, SDL Core selects one of the status transition rules with each situation.
```
Please also specify which SDL Component you're referencing in "When the SDL is launched".  Should this be "The SDL app"?

Reply:
Hi, @theresalech -san.
Thank you for your review.
I understood. I will modify this sentence to "When the SDL app is launched," and make a commit it.
Could you please confirm it again.
Thank you.